File: array & sorting/_EASY_2037.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int minMovesToSeat(vector<int>& seats, vector<int>& students) {
       sort(seats.begin(), seats.end());
       sort(students.begin(), students.end());
        int moves = 0;
        for( int i=0; i<seats.size(); i++)
            moves += abs(seats[i] - students[i]);
        
        return moves;
    }
};

int main(){
    Solution s;
    vector<int> seats = {4,3,6,5};
    vector<int> students = {5,2,1,8};
    cout<<s.minMovesToSeat(seats, students)<<endl;
    return 0;
}

File: array & sorting/_EASY_2037.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
var minMovesToSeat = function(seats, students) {
    let moves = 0
    seats = seats.sort((a, b) => b - a)
    students = students.sort((a, b) => b - a)
    seats.forEach((s, i) =>
        moves = moves + Math.abs(s - students[i]))

    return moves
};

function main() {
    let seats = [4, 3, 8, 6]
    let students = [5, 2, 6, 1]
    let result = minMovesToSeat(seats, students)
    console.log(result)
}

main()